src/ble/BleConfig.h

```diff
@@ -112,6 +112,16 @@
 #define BLE_CONFIG_BLUEZ_MTU_FEATURE 0
 #endif // BLE_CONFIG_BLUEZ_MTU_FEATURE
 
+/**
+ *  @def BLE_CONFIG_CROS_BLUEZ_FEATURE
+ *
+ *  @brief
+ *    This define if CROS BLUEZ FEATURE is enabled or not
```

Comment:
When you say "CROS BLUEZ" do you mean ChromeOS's implementation of BlueZ?  If so, please make that clear in the description.

Reply:
updated description, thanks,